### PR TITLE
Model lending was added and relations with model user.

### PR DESCRIPTION
**Pull request by miguelhuchim05**

### CODE REVIEW

File: app/models/lending.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Lending
+  include Mongoid::Document
+  include Mongoid::Enum
+  rolify
+
+  field :start_date, type: Date
+  field :end_date, type: Date
+  field :reason, type: String
+  enum :status, %i[started finished]
```
Comment (TonyBM) [inline app/models/lending.rb]: Ponlo abajo de los fields

File: app/models/lending.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Lending
+  include Mongoid::Document
+  include Mongoid::Enum
+  rolify
+
+  field :start_date, type: Date
+  field :end_date, type: Date
+  field :reason, type: String
+  field :user_id, type: String
+  field :authorized_by, type: String
```
Comment (Pako7) [inline app/models/lending.rb]: User

File: app/models/lending.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class Lending
+  include Mongoid::Document
+  include Mongoid::Enum
+  rolify
+
+  field :start_date, type: Date
+  field :end_date, type: Date
+  field :reason, type: String
+  field :user_id, type: String
```
Comment (Pako7) [inline app/models/lending.rb]: User

File: app/models/lending.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  field :authorized_by, type: String
+  enum :status, %i[started finished]
+  
+  belongs_to :user
+
+  validates_presence_of :start_date, :end_date, :reason, :status, :user_id, :authorized_by
```
Comment (Pako7) [inline app/models/lending.rb]: No end_date

**Pull request merged**